test/com/google/cose/TestUtilities.java

```diff
@@ -0,0 +1,26 @@
+package com.google.cose;
+
+public class TestUtilities {
+  public static final String CONTENT = "This is the content.";
```

Comment:
nit: I think `"This is the content."` as a literal is actually more readable. Up to you if you want to change it.